### ISSUE TITLE
Check if Mappings for CLS (and probably some were else) are right

### ISSUE THREAD

**Issue by christianmahnke** [subugoe/goefis#98]
This is not a blocker for the staging.

We need to check if the document type mappings for citations are correct, currently document 7, is returned as a 'book', see:
http://pub-dev.sub.uni-goettingen.de/livecitation?id=7&style=ama&debug=1

**Comment by njahn82** [comment]
Good news, I installed the docker dev image and the mapping works as intended. I guess we simply need to update the records, so that the new mappings become productive.

**Comment by christianmahnke** [comment]
Bad news: In 0.3.1 this breaks:
https://github.com/LibreCat/LibreCat/issues/168

**Comment by njahn82** [comment]
Played around with the patch workflow today. Because I can access the
sources and error logs via Docker we can try our best without it (there are
many citeproc error messages, I don't understand)

Am 17.02.2017 4:43 nachm. schrieb "christianmahnke" <
notifications@github.com>:

**Comment by christianmahnke** [comment]
My proposal ist to postpone it a week or so and hope they fix it. Or do you think we need it for staging? Or do you need it? I can give you such a server with memory caching, it should be really fast, it's here:
https://github.com/subugoe/goefis-datatools/tree/master/citeproc-nginx

**Comment by njahn82** [comment]
Yes, the livecitation is a super minor. We have some styles working, and I
will submit a ticket asking them to provide guidance how to add styles.

**Comment by njahn82** [comment]
We need to add an language field, so that non-english titles are not displayed in title case
http://docs.citationstyles.org/en/stable/specification.html#non-english-items

**Comment by christianmahnke** [comment]
Is this something, that would improve LibreCat? Something that they can benefit from as well?

**Comment by njahn82** [comment]
Yeah, guess I need to add something to the fix file. Will try and if
succesful create a PR for LibreCat core

Am 22.03.2017 5:52 nachm. schrieb "christianmahnke" <
notifications@github.com>:

**Comment by njahn82** [comment]
https://github.com/LibreCat/LibreCat/issues/224
